airbyte-integrations/connectors/source-github/source_github/schemas/stargazers.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "_sdc_repository": {
+      "type": ["string"]
+    },
+    "user_id": {
+      "type": ["null", "integer"]
+    },
+    "starred_at": {
```

Comment:
```suggestion
    "started_at": {
```

Reply:
It's not a typo.
![image](https://user-images.githubusercontent.com/19872253/124136843-d09b1380-da8d-11eb-82f2-e4cc82e84c72.png)